package.json

```diff
@@ -51,11 +52,16 @@
     "long": "^4.0.0",
     "md5": "^2.2.1",
     "pako": "^1.0.4",
-    "quick-lru": "^2.0.0"
+    "quick-lru": "^6.1.1"
```

Comment:
might suggest dropping this back down and using @types/quick-lru. the newer version uses too many new js constructs and requires e.g. the transformignorepatterns above which basically implies consumers of cram-js would feel this too

Reply:
v2 are suspiciously not used. updated to v4, which should still be backward compatible, but already have integrated correct typing (v2 was causing typing errors somewhere)